products/bridge/test/utils.ts

```diff
@@ -38,12 +104,9 @@ export async function switchNetwork(id = 0) {
 export async function obtainCalls(validators: Signer[], relayer: Relayer) {
   // validators see the Relayed event
   const randIndex = Math.floor(Math.random() * validators.length);
-  const blockNum = await ethers.provider.getBlockNumber();
   const filter = relayer.filters.Relayed;
   // Select random validator to query (in tests they use the same provider, but not in reality)
-  const logs = await relayer
-    .connect(validators[randIndex])
-    .queryFilter(filter, blockNum - 100, blockNum);
+  const logs = await relayer.connect(validators[randIndex]).queryFilter(filter);
```

Comment:
As discussed, it's essential to only consider events in finalized blocks, therefore the code before refactoring fetched `ethers.provider.getBlockNumber('finalized')` and queried only until that block number.